reconcile/openshift_base.py

```diff
@@ -252,6 +253,24 @@ def apply(dry_run, oc_map, cluster, namespace, resource_type, resource,
             oc.delete(namespace=namespace, kind=resource_type,
                       name=resource.name)
             oc.apply(namespace=namespace, resource=annotated)
+        except StatefulSetUpdateForbidden:
+            if resource_type not in ['StatefulSet']:
+                raise
+
+            logging.info(['delete_and_apply', cluster, namespace,
```

Comment:
nit: since you know where you are, I would make this more explicit so it's easier to find the place where this is written when debugging something like this
```suggestion
            logging.info(['delete_sts_and_apply', cluster, namespace,
```